tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -349,6 +349,7 @@ def run_func(self):
 
 class WebDriverTestharnessExecutor(TestharnessExecutor):
     supports_testdriver = True
+    protocol_cls = WebDriverProtocol
```

Comment:
I don't understand what these changes are about; can you explain?

Reply:
It's basically a mechanism to allow the `ChromeDriverPrintReftestExecutor` to inherit from the `WebDriverReftestExector` but also have a special `ChromeDriverProtocol` which is an extension of the `WebDriverProtocol` including the CDP-implemented print parts.

More abstractly it makes the protocol a swappable component rather that an intrinsic part of the executor.